backend/models/institution.py

```diff
@@ -59,6 +59,8 @@ class Institution(ndb.Model):
     # The admin user of this institution
     admin = ndb.KeyProperty(kind="User")
```

Comment:
why the model still has to maintain the admin property?

Reply:
@JuliePessoa he can't remove it now, if he do that, he will face many problems

Reply:
So that the modifications can be divided into tasks without breaking the other features of the system, eventually all the features will be modified and the admin attribute will no longer be necessary and can be removed.